backend/routes/authRouter.js

```diff
@@ -2,43 +2,56 @@ import { Router } from "express";
 import authHelpers from "../helpers/auth";
 
 export default (db) => {
+    const { createUser, findUserById, validateUser } = authHelpers(db);
+    const auth = Router();
 
-	const { createUser, findUserById, validateUser } = authHelpers(db)
-	const auth = Router();
+    auth.post("/validation", (req, res, next) => {
+        findUserById(req.session.userId)
+            .then((user) => {
+                const { name, email, _id, avatarImage, recipes } = user;
+                res.json({
+                    name,
+                    email,
+                    islanderId: _id,
+                    avatarImage,
+                    recipes,
+                });
+            })
+            .catch(() => res.status(403).send("Invalid request"));
+    });
 
-	auth.post("/validation", (req, res, next) => {
-		findUserById(req.session.userId)
-			.then(user => {
-				const { name, email, _id, avatarImage, recipes } = user;
-				res.json({ name, email, islanderId: _id, avatarImage, recipes });
+    auth.post("/login", (req, res, next) => {
+        validateUser(req.body.email, req.body.password)
+            .then((user) => {
+                console.log("User validated...");
+                console.log(user);
+                req.session.userId = user._id;
+                const { name, email, _id, avatarImage, recipes } = user;
+                res.json({
+                    name,
+                    email,
+                    islanderId: _id,
+                    avatarImage,
+                    recipes,
+                });
+            })
+            .catch((err) => {
+                console.log("Error during validation:", err);
+                res.status(403).json({});
+            });
+    });
 
-			})
-			.catch(() => res.status(403).send("Invalid request"));
-	});
+    auth.post("/register", (req, res, next) => {
+        console.log(req.body);
+        const { name, email, password, islandId } = req.body;
+        const user = createUser({ name, email, password, islandId });

```

Comment:
If `createUser` throws and error, is that error handled?

Reply:
I didn't actually make any changes here - linting and format on save caused this to change drastically. Toggle on "Hide whitespace changes" and that should show it.

That being said, there's no error handling as I found out through testing this 😅 Let's come back to this as we refactor and cleanup the Registration page